_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -34,7 +34,7 @@ Each test target's `autocomplete` [attribute value][] is a [space separated][] l
 
 1. An optional token that starts with "section-"; then
 2. An optional token of either "shipping" or "billing"; then
-3. An optional token of either "home", "work", "mobile", "fax" or "pager", only if the last token is "email", "impp", "tel" or "tel-*"; then
+3. An optional token of either "home", "work", "mobile", "fax" or "pager", only if the last token is "email", "impp", "tel" or "tel-\*"; then
```

Comment:
I'm not sure why you add a backslash here 🤔 
OTOH, maybe all these tokens (also in other items of the list) might need to be in backticks as they are essentially code.

Reply:
Hey @Jym77 

When I first edited the file, the editor came back with a syntax error so I thought that I needed to escape it. Will change it now.